activerecord/lib/active_record/persistence.rb

```diff
@@ -280,15 +280,26 @@ def instantiate(attributes, column_types = {}, &block)
       #   people.update(group: "masters")
       #
       # Note: Updating a large number of records will run an UPDATE
-      # query for each record, which may cause a performance issue.
+      # query for each record(duplicates will be merged together first),
+      # which may cause a performance issue.
```

Comment:
Since this sentence is about performance, it may make more sense to keep it as a different sentence and explain how the merge is done.

```suggestion
      # query for each record which may cause a performance issue.
      # Duplicate ids will only be updated once, by merging all attributes.
```